scripts/sd_tag_batch.py

```diff
@@ -40,17 +83,25 @@ def run(self, p, in_front, prompt_weight, use_deepbooru, use_weight):
             interrogator = deepbooru.model.tag(p.init_images[0])
         else:
             interrogator = shared.interrogator.interrogate(p.init_images[0])
+        
+        # Remove duplicate prompt content from interrogator prompt
+        if no_duplicates:
+            interrogator = self.filter_words(interrogator, raw_prompt)
+        # Remove negative prompt content from interrogator prompt
+        if use_negatives:
+            raw_negative = p.negative_prompt
+            interrogator = self.filter_words(interrogator, raw_negative)
```

Comment:
Unsure if there was the possibility of pointer override, so I added a buffer variable. 
I am not sure if this is an issue to think about in Python...

Reply:
I just learned that Python uses the "pass by assignment" concept. So pointer override is not something to worry about, so https://github.com/Alvi-alvarez/sd-Img2img-batch-interrogator/pull/6/commits/8412f15aa90369fe10555f745ec8a6c22cdba3a8 should be used.